### ISSUE TITLE
republish docs

### ISSUE THREAD

**Issue by Gankra** [volks73/cargo-wix#265]
there's docs on the new configs added in the last release, needs a deploy.

**Comment by volks73** [comment]
I thought I did republish the docs to https://volks73.github.io/cargo-wix/cargo_wix when I created the most recent release, v0.3.8. I did `cargo xtask ghp`, which is supposed to automatically run rustdoc and push the changes to GitHub Pages. It is possible that failed in some hidden way.

Is the missing configs docs for the library or the binary (subcommand)? I believe the library documentation is handled through docs.rs, but I could be wrong on this.

Which configs docs are missing from the published docs? I would like to debug a little the xtask or docs.rs.

**Comment by Gankra** [comment]
Oh shit, you did! Thanks!

I could have sworn it wasn't updated yesterday, but maybe... caching..?

**Comment by Gankra** [comment]
![image](https://github.com/volks73/cargo-wix/assets/1136864/53417ccc-bc88-47dc-a900-afc8be0b39e9)

Ah i see what messed me up. Yesterday I went to the docs specifically to make sure banner/dialog/product-icon were properly listed here, and I could have sworn I added them, but, I guess I lost it in a rebase or something (the new docs do have my updates to the individual *entries* for those, so I did land the most important docs).

Oh well, I'll submit a followup pr to add those.
